persistence/src/main/java/org/mskcc/cmo/metadb/persistence/MetaDbSampleRepository.java

```diff
@@ -72,4 +72,8 @@ MetaDbSample findMetaDbSampleByRequestAndIgoId(@Param("reqId") String reqId,
             + "RETURN sm"
     )
     List<SampleMetadata> findSampleMetadataListByCmoPatientId(@Param("cmoPatientId") String cmoPatientId);
+    
+    @Query("MATCH (sm:SampleMetadata {cmoInfoIgoId: $igoSampleId})"
```

Comment:
something else I just noticed: LIMS sends two really similar fields - `cmoInfoIgoId` and `igoId` in a given sample's manifest

we consistency refer to `igoId` though and not `cmoInfoIgoId` so the param here should prob change to be consistent as well as the param name `$igoSampleId`